docs/docs/documentation/getting-started/quick-start.md

```diff
@@ -9,16 +9,53 @@ sidebar_position: 30
 Looking to get started with plugins? For more information, see the [Plugins](../concepts/plugins.md).
 :::
 
-To get started with Solace Agent Mesh, you must first create a project.
+To get started with Solace Agent Mesh, you can either create a complete project or use the preset configuration from the Docker image.
 
-## Prerequisites
+## Quick Start
 
-1. You have installed the Solace Agent Mesh CLI. If not, see the [Installation](./installation.md) page.
-2. You have activated the virtual environment you created following the [Installation](./installation.md) page. For containerized deployment such as Docker, ignore this prerequisite.
-3. You have an available AI provider and API key. For best results, use a state-of-the-art AI model like Anthropic Claude Sonnet 3.7, Google Gemini 2.5 pro, or OpenAI GPT-4o.
+### Prerequisites
+1. You have Docker (or Podman) installed.
+2. You have an available AI provider and API key. For best results, use a state-of-the-art AI model like Anthropic Claude Sonnet 4, Google Gemini 2.5 pro, or OpenAI GPT-5.
+
+### Run The Preset App
+
+You can run pre-made SAM applications to quickly get started with Solace Agent Mesh without needing to set up a project from scratch. 
+
+```sh
+docker run --rm -it -p 8000:8000 --platform linux/amd64  --env-file <your-env-file-path> solace/solace-agent-mesh:latest
+```
+
+- Provide the required environment variables. You can also pass them as command-line arguments using the `-e` flag.
+
+Alternatively, you can combine your project with the preset app or only run specific agents from it.
+
+You can find a list of all available preset apps in the [Solace Agent Mesh GitHub repository](https://github.com/SolaceLabs/solace-agent-mesh/tree/main/preset/agents).
+
+:::tip
+To only run your custom agent without creating a new project, you can use the Docker image as follows:
+
+```bash
+docker run --rm -it --platform linux/amd64  -p 8000:8000 -v $(pwd):/app \
+  -e LLM_SERVICE_ENDPOINT=<your-llm-endpoint> \
+  -e LLM_SERVICE_API_KEY=<your-llm-api-key> \
+  -e LLM_SERVICE_PLANNING_MODEL_NAME=<your-llm-planning-model-name> \
+  -e LLM_SERVICE_GENERAL_MODEL_NAME=<your-llm-general-model-name> \
+  solace/solace-agent-mesh:latest run /preset/agents/basic /app/my-agent
+```
+
+- You can update `/app/my-agent` with the path to your agent yaml config. (Note: you'd still need `shared_config.yaml` file or have to hard-code it in your agent config)
```

Comment:
Two notes here:

- Capitalize YAML unless it's part of a named file.

- Reword your note and add a period. You don't need parentheses here. "Note: You will still need either a `shared_config.yaml` file or to hard code the settings in your agent configuration."